pyannote/audio/features/with_librosa.py

```diff
@@ -156,19 +156,24 @@ def get_features(self, y, sample_rate):
             Waveform
         sample_rate : int
             Sample rate
+        epoch :
+            Current epoch
 
         Returns
         -------
         data : (n_frames, n_mels) numpy array
             Features
         """
-
-        X = librosa.feature.melspectrogram(
+        mel_spec = librosa.feature.melspectrogram(
             y.squeeze(), sr=sample_rate, n_mels=self.n_mels,
-            n_fft=self.n_fft_, hop_length=self.hop_length_,
-            power=2.0)
+            n_fft=self.n_fft_, hop_length=self.hop_length_, power=2)
+
+        mel_spec = librosa.power_to_db(mel_spec, ref=np.max)
+
+        if self.norm:
+            mel_spec = (mel_spec - np.mean(mel_spec)) / np.var(mel_spec)
```

Comment:
I'd rather have the normalization stuff directly in the first layer of the model (hence in the GPU)
Any reason why this would have to be done here (and therefore on CPU)?